.github/workflows/ci.yml

```diff
@@ -0,0 +1,181 @@
+name: Build and push docker images
+
+on: [push, pull_request]
+
+# Version number of the docker images - increment when there are huge changes
+env:
+  VERSION_NUMBER: '1.0.13'
+  LATEST_PYTHON_VERSION: '3.9'
+  LATEST_SC2_VERSION: '4.10'
+
+# TODO Enable python-version '3.10' in all workflow matrices when https://github.com/aiarena/rust-arenaclient/pull/37 was merged
```

Comment:
I've requested a review from Dan on this PR.

Reply:
He should be aware but just doesn't seem to have the time at the moment